src/transformers/models/univnet/modeling_univnet.py

```diff
@@ -0,0 +1,692 @@
+        spectrograms returns a batch of speech waveforms. Passing a single, un-batched log-mel spectrogram returns a
+        single, un-batched speech waveform.
+
+        Args:
+            spectrogram (`torch.FloatTensor`):
+                Tensor containing the log-mel spectrograms. Can be batched and of shape `(batch_size, sequence_length,
+                config.num_mel_channels)`, or un-batched and of shape `(sequence_length, config.num_mel_channels)`.
+            noise_waveform (`torch.FloatTensor`, *optional*):
+                Tensor containing a noise waveform sequence of standard Gaussian noise. Can be batched and of shape
+                `(batch_size, sequence_length, config.model_in_channels)`, or un-batched and of shape (sequence_length,
+                config.model_in_channels)`. If not supplied, will be randomly generated.
+            generator (`torch.Generator`, *optional*):
+                A [torch generator](https://pytorch.org/docs/stable/generated/torch.Generator.html) to make generation
+                deterministic.
+
+        Returns:
+            `torch.FloatTensor`: Tensor containing the speech waveform. IF the input spectrogram is batched, wil be of
+            shape `(batch_size, num_frames)`. If un-batched, will be of shape `(num_frames,)`.
+
+        Example:
+
+         ```python
+         >>> import torch
+         >>> from transformers import UnivNetFeatureExtractor, UnivNetGan
+         >>> from datasets import load_dataset, Audio
+
+         >>> model = UnivNetGan.from_pretrained("dg845/univnet-dev")
+         >>> feature_extractor = UnivNetFeatureExtractor.from_pretrained("dg845/univnet-dev")
+
+         >>> ds = load_dataset("hf-internal-testing/librispeech_asr_dummy", "clean", split="validation")
+         >>> # Resample the audio to the feature extractor's sampling rate.
+         >>> ds = ds.cast_column("audio", Audio(sampling_rate=feature_extractor.sampling_rate))
+         >>> inputs = feature_extractor(
+         ...     ds[0]["audio"]["array"], sampling_rate=ds[0]["audio"]["sampling_rate"], return_tensors="pt"
+         ... )
+         >>> input_features = inputs.input_features.float()
+
+         >>> # Padding the input spectrogram with "zeros" reduces artifacts at the end of the generated sample.
+         >>> # See https://github.com/seungwonpark/melgan/issues/8
+         >>> pad_len = 10
+         >>> padding_zeros = torch.full((1, pad_len, model.config.num_mel_channels), -11.5129)
+         >>> input_features = torch.cat([input_features, padding_zeros], dim=1)
+         >>> # UnivNetGan also takes in standard Gaussian noise as input. If the noise argument is not supplied, it
+         >>> # be sampled inside the model call.
+         >>> noise = torch.randn((1, input_features.shape[1], model.config.model_in_channels))
+         >>> audio = model(input_features, noise)  # audio = model(input_features) also works
```

Comment:
It should be possible to use `audio = model(**inputs)` where `inputs` is the output of the feature extractor. I have set the feature extractor to not return an `attention_mask` since there's no way for `UnivNetGan` to use it.